src/rules/no-disabled-tests.ts

```diff
@@ -1,5 +1,4 @@
-import { createRule } from './tsUtils';
-import { getNodeName, scopeHasLocalReference } from './tsUtils';
+import { createRule, getNodeName, scopeHasLocalReference } from './tsUtils';
```

Comment:
landed this on master btw, along with an (autofixable) eslint rule so it won't sneak in again

Reply:
This can be rebased away :D